core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -238,6 +244,13 @@ oppia.directive('audioTranslationBar', [
               $scope.initAudioBar();
             });
           };
+          var shortcutkeyForRecording = function() {
```

Comment:
rename this to toggleStartAndStopRecording.

Reply:
Done